src/main/resources/assets/tconstruct/book/materials_and_you/index.json

```diff
@@ -1,10 +1,10 @@
 [
   {
     "name": "intro",
-    "data": "sections/intro_tmp.json",
+    "data": "sections/introduction.json",
     "icon": {
       "item": {
-        "id": "tconstruct:blue_slime_ball"
+        "id": "tconstruct:pattern"
```

Comment:
Change the pickaxe for tools to a Tinkers' pickaxe, using the render materials we showed you